app-admin/go-updater/go-updater-0.2.0-r1.ebuild

```diff
@@ -10,15 +10,12 @@ SRC_URI="https://github.com/mrueg/go-updater/archive/${PV}.tar.gz -> ${P}.tar.gz
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64"
-IUSE=""
 
 RDEPEND="dev-go/goversion"
 
-src_compile() {
-	:;
-}
+src_compile() { :; }
 
 src_install() {
-	dobin ${PN}
+	dobin "${PN}"
```

Comment:
see earlier quoting comments although this can be okay I guess under the guideline I mentioned